### ISSUE TITLE
Ability of a user to access Organization#index

### ISSUE THREAD

**Issue by rishabhptr** [openSUSE/osem#1934]
**I'm submitting a ..**
- [x] Bug Report

**Current behavior:**
Now that there is a organization logo in the admin sidebar, it makes sense the authorisation of organization#index is changed as right now users which are volunteers,organizers, have a accepted track request etc all can see that logo and though cant perform any action there they are still able to view all org admins throughout the application.
![screenshot from 2018-01-05 23-31-13](https://user-images.githubusercontent.com/26951824/34621797-8aa4742e-f270-11e7-83b6-63c46488f7e6.png)

**Comment by differentreality** [comment]
I am not sure I understand the problem.
Anyone with access to a part of the admin side can see who is an organization admin. What's wrong with that? 

That said, there are a couple of other things worth fixing regarding this (and previous) page.

**Comment by rishabhptr** [comment]
@differentreality Yes if think we could check abilities of users in organization#index same as we do for Roles.

**Comment by differentreality** [comment]
I am afraid I still don't understand what you are aiming at. Please elaborate on your idea.

**Comment by rishabhptr** [comment]
I meant to say if users with all role(cfp,volunteer etc ) are given access to organization#index then we could check their ability and only show buttons for action they can perform.
Because right now only organization admin can edit his org(cant create/destroy) and all other roles can view admins.
So by doing this organization admins will see edit/destroy button for his org and all other roles would see only admin button.
